packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -417,6 +450,10 @@ class MessageList extends Widget<Properties> {
       case 'showDayHeaders':
         this._invalidate();
         break;
+      case 'typingUsers':
+        this._typingIndicator.option(name, value);
+        this._scrollDownContent();
```

Comment:
I am not sure that we need scrolldown every time when typingUsers is changed. For example, I can scroll up to check Chat's history and when somebody starts type my scroll will change.

Reply:
For me it also looks redundant and not expected

Reply:
Fix: [47e80e6de3051c8f384466525f90a19ffcb0560a]
Tests: [a714680ada5a7e97314efe9b0168c6f6dd53ac8b]